package.json

```diff
@@ -79,10 +79,6 @@
       },
       "locator": "cordova-plugin-crosswalk-webview",
       "id": "cordova-plugin-crosswalk-webview"
-    },
-    {
-      "locator": "../me.apla.cordova.app-preferences",
```

Comment:
nit: you don't want to replace this completely - just with a reference to the original appPreference.